components/brave_wallet_ui/components/shared/add-custom-token-form/add-custom-token-form.tsx

```diff
@@ -152,6 +152,7 @@ export const AddCustomTokenForm = (props: Props) => {
         coingeckoId,
         logo: iconURL,
         tokenId: '',
+        isCompressed: false,
```

Comment:
Do we need to add a lookup query to determine this value?

Reply:
Only NFTs will be compressed, so assuming this form does not handle NFTs, false will always be correct here. If they are NFTs, the backend will fetch and set the correct value from simplehash